modules/TimeEngine.py

```diff
@@ -6,35 +6,34 @@
 
 # Importation des librairies.
 from datetime import datetime
-from modules.ConfigEngine import get_config, get_config_boolean
+from modules.ConfigEngine import BaseConfigEngine
 
 
 def get_days():
     containment_date = datetime(2020, 3, 17)  # Confinement
     today_date = datetime.today()
 
-    number_of_days = str(today_date - containment_date)
-    number_of_days = str([int(s) for s in number_of_days.split() if s.isdigit()])
-    number_of_days = number_of_days.replace('[', '')
-    number_of_days = number_of_days.replace(']', '')
-    number_of_days = int(number_of_days) + 1  # Le confinement a débuté à midi.
-
-    return str(number_of_days)
+    # Le confinement a débuté à midi.
+    return str((today_date - containment_date).days - 1)
```

Comment:
Big optimisation here. You can simply use  ```days``` instead of parsing the return string !